dom_sink/src/owned_dom.rs

```diff
@@ -197,6 +197,17 @@ impl TreeSink for Sink {
         x == y
     }
 
+    fn same_home_subtree(&self, _x: Handle, _y: Handle) -> bool {
+        true
+    }
+
+    fn associate_with_form(&mut self, _target: Handle, _form: Handle) {
+    }
```

Comment:
It can be useful to parse HTML in a "real" application that is not a browser doesn't support form submission.